reminders/models.py

```diff
@@ -0,0 +1,48 @@
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+from django.db import models
+from events.models import EventParticipant
+
+
+def validate_date(date_time):
+    if timezone.now() > date_time:
+        raise ValidationError(f'{time_format(date_time)} is not a valid date_time')
```

Comment:
maybe it will be more informative like:
```suggestion
        raise ValidationError(f'{time_format(date_time)} should be bigger than current date_time')
```